### DISCUSSION TITLE
Algolia and Meilisearch as SCOUT_DRIVER

### DISCUSSION THREAD

**Discussion by nick-nds** [lunarphp/lunar#1099] [Feature Requests]
Hello,
On the installation page, it is said to use `database_index` as SCOUT_DRIVER. I was hoping that Lunar would support other drivers supported by Laravel out of the box, such as [Algolia](https://laravel.com/docs/10.x/scout#algolia) and [Meilisearch](https://laravel.com/docs/10.x/scout#meilisearch).

**Comment by glennjacobs** [comment]
You can use any Scout driver you like. We use Algolia, Typesense and Meilisesrch.

**Reply by nick-nds** [reply]
Thank you
